packages/neuron-wallet/src/exceptions/wallet.ts

```diff
@@ -78,6 +78,13 @@ export class WalletFunctionNotSupported extends Error {
   }
 }
 
+export class ImportingExitingWallet extends Error {
```

Comment:
The `Exiting` may be `Existing`. I suggest the name `DuplicateImportWallet` for easy understanding